.changes/unreleased/Under the Hood-20220905-001749.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Under the Hood
+body: Migrate integration test 014 but also make a few testing suite fixes
```

Comment:
let's expand on "fixes" here to be more specific in case we need to find this commit again.